apps/dc_tools/odc/apps/dc_tools/export_md.py

```diff
@@ -0,0 +1,264 @@
+"""
+Metadata Transformation from old format to new format:
+Example usage:
+ python metadata_transformer.py --datacube-config $HOME/.datacube.conf transform --product aster_l1t_swir
+    --config metadata_transform_config.yaml --output-dir /g/data/u46/users/aj9439/metadata
+    --limit 10
+The config file contains the grid mappings to band names:
+    grids:
+      default: ['3', '4', '5']
+      swir: ['1', '2']
+"""
+from pathlib import Path
+import yaml
+import click
+
+from datacube.testutils.io import native_geobox
+from datacube import Datacube
+
+
+@click.group(help=__doc__)
+@click.option('--datacube-config', '-c', help="Pass the configuration file to access the database",
+              type=click.Path(exists=True))
+@click.pass_context
+def cli(ctx, datacube_config):
+    """ Used to pass the datacube index to functions via click."""
```

Comment:
This string is used by `click` to provide `--help` output, so it needs to be user friendly.

Reply:
Done